crypto/evp/evp_fetch.c

```diff
@@ -85,8 +85,8 @@ static OSSL_METHOD_STORE *get_evp_method_store(OPENSSL_CTX *libctx)
  */
 static uint32_t evp_method_id(unsigned int operation_id, int name_id)
```

Comment:
BTW: @levitte I am a little bit irritated about the order of the function parameters `name_id` and `operation_id`, which  is reverted compared to their order of appearance in the comment and the assertions. 

https://github.com/openssl/openssl/blob/55164d68086e534e421ec4ac9753982114664358/crypto/evp/evp_fetch.c#L75-L86

Reply:
Feel free to change it

Reply:
Done, see c94bbd7fe661610dcc1c45c5258b4090289adaf3.

Reply:
Ups, there was a mistake.  Commit replaced by 13367d3e947e205350c5a46a27cac8466f9bb3a7.